src/iconthemewrapper.cpp

```diff
@@ -27,3 +27,31 @@ procman::IconThemeWrapper::load_icon(const Glib::ustring& icon_name, int size) c
     }
 }
 
+Glib::RefPtr<Gdk::Pixbuf>
+procman::IconThemeWrapper::load_gicon(const Glib::RefPtr<Gio::Icon>& gicon,
+                                      int size, Gtk::IconLookupFlags flags) const
+{
+    Gtk::IconInfo icon_info;
+    gint scale = gdk_window_get_scale_factor (gdk_get_default_root_window ());
+    icon_info = Gtk::IconTheme::get_default()->lookup_icon(gicon, size, scale, flags);
+
+    if (!icon_info) {
+        return Glib::RefPtr<Gdk::Pixbuf>();
+    }
+
+    try
```

Comment:
Build warning.
```
iconthemewrapper.cpp:42:5: warning: This statement is never executed

    try

    ^~~
```
https://travis-ci.org/github/mate-desktop/mate-system-monitor/jobs/711714681#L2894

Reply:
It's a false positive because it overrides the boolean operator.

> IconInfo::operator bool() will return false if the icon wasn't found. For instance, if (icon_info) { ... }.

https://developer.gnome.org/gtkmm/stable/classGtk_1_1IconTheme.html#a65eb6be589a121ece1bc5b267a033ab5

Reply:
OK